x-pack/plugins/canvas/canvas_plugin_src/functions/browser/urlparam.ts

```diff
@@ -43,7 +45,9 @@ export function urlparam(): ExpressionFunctionDefinition<
       },
     },
     fn: (input, args) => {
-      const query = parse(window.location.href, true).query;
+      const uri = parse(window.location.href, true);
+      const viewParam = qs.parse((uri.hash.includes("?")) ? uri.hash.substring(uri.hash.indexOf("?"),uri.hash.length) : "");
```

Comment:
This is not formatted properly (`"` instead of `'`)... do you have `eslint` enabled?